core/components/atoms/tooltip/tooltip.tsx

```diff
@@ -1,58 +1,60 @@
-import * as React from 'react'
-import styled from '../../styled'
-import { Manager, Reference, Popper, RefHandler } from 'react-popper'
-import Automation from '../../_helpers/automation-attribute'
-import { multiply } from '../../_helpers/pixel-calc'
-import uniqueId from '../../_helpers/uniqueId'
+import * as React from "react";
+import { Manager, Popper, Reference, RefHandler } from "react-popper";
 
-import { colors, spacing, misc } from '../../tokens'
+import Automation from "../../_helpers/automation-attribute";
+import { multiply } from "../../_helpers/pixel-calc";
+import uniqueId from "../../_helpers/uniqueId";
+import styled from "../../styled";
+import { colors, misc, spacing } from "../../tokens";
 
-const arrowWidth = '6px'
-const arrowColor = colors.tooltip.background
+const arrowWidth = "6px";
+const arrowColor = colors.tooltip.background;
 
 /*
   Popper doesn't arrange the tooltip right in the center,
   so we add an adjustment
 */
-const arrowAdjustment = multiply(arrowWidth, -2)
+const arrowAdjustment = multiply(arrowWidth, -2);
 
-export type TooltipPosition = 'top' | 'bottom' | 'left' | 'right'
+export type TooltipPosition = "top" | "bottom" | "left" | "right";
 export interface ITooltipProps {
   /** HTML ID of the component */
-  id?: string
+  id?: string;
   /** Content to show in the tooltip */
-  content: string
+  content: string;
   /** Where to place the tooltip */
-  position?: TooltipPosition
+  position?: TooltipPosition;
   /** Visible by default */
-  defaultVisible?: boolean
-  onClick?: Function
+  defaultVisible?: boolean;
+  onClick?: Function;
 }
 
 interface ITooltipState {
-  visible: boolean
-  id: string
+  visible: boolean;
+  id: string;
 }
 
 interface IObservedElementProps {
-  scheduleUpdate: Function
-  innerRef: RefHandler
-  style: Object
-  'data-placement': string
-  id: string
+  scheduleUpdate: Function;
+  innerRef: RefHandler;
+  style: Object;
+  "data-placement": string;
+  id: string;
 }
 
 class Tooltip extends React.Component<ITooltipProps, ITooltipState> {
   public static ObservedElement = class extends React.Component<IObservedElementProps> {
     public componentDidUpdate() {
-      this.props.scheduleUpdate()
+      this.props.scheduleUpdate();
     }
     public render() {
-      return <Tooltip.Element {...this.props} />
+      const { innerRef, ...props } = this.props;
+
+      return <Tooltip.Element {...props} ref={innerRef} />;
```

Comment:
Change here. I kept innerRef for now, but we can switch to forwardRef in a followup.